app/src/main/java/eu/faircode/netguard/Receiver.java

```diff
@@ -224,13 +224,9 @@ public static void upgrade(boolean initialized, Context context) {
                 Log.i(TAG, "Initializing sdk=" + Build.VERSION.SDK_INT);
                 editor.putBoolean("whitelist_wifi", false);
                 editor.putBoolean("whitelist_other", false);
-                if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
So, bug, because filtering should be enabled on 5.0, not 5.1
I don't remember what was wrong with the Note 4, can you refresh my memory?